devices.js

```diff
@@ -3793,9 +3793,16 @@ const devices = [
             const endpoint = device.getEndpoint(1);
             await bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
             await configureReporting.onOff(endpoint);
-            await configureReporting.brightness(endpoint);
+            const payload = [{
+              attribute: 'currentLevel',
+              minimumReportInterval: 300,
```

Comment:
Does it still work when changing this back to 0? (don't forget to bump the `configureKey` to `2` after this change)